### ISSUE TITLE
pimd with `--enable-scoped-acls` FTBS

### ISSUE THREAD

**Issue by greenpau** [troglobit/pimd#40]
When compiling `pimd-2.2.0` with `--enable-scoped-acls`,  I received the following error: 

```
# ./configure --enable-scoped-acls
# make
  CC      route.o
route.c: In function ‘process_cache_miss’:
route.c:90:14: error: assignment makes integer from pointer without a cast [-Werror]
   (mp)->oifs = NULL;  \
              ^
route.c:906:6: note: in expansion of macro ‘APPLY_SCOPE’
      APPLY_SCOPE(group,mrt);
      ^
route.c:90:14: error: assignment makes integer from pointer without a cast [-Werror]
   (mp)->oifs = NULL;  \
              ^
route.c:938:3: note: in expansion of macro ‘APPLY_SCOPE’
   APPLY_SCOPE(group,mrt);
   ^
cc1: all warnings being treated as errors
make: *** [route.o] Error 1
#
```

Without the flag, I am able to build it successfully.

```
# pimd -v
pimd version 2.2.0
#
```

Please assist.

**Comment by troglobit** [comment]
Try the latest GIT master, that should work.

Fixed in git commit 2a4ee1a for the upcoming v2.2.2 release.

**Comment by greenpau** [comment]
@troglobit , thank you!
